src/main/java/pika/ui/Storage.java

```diff
@@ -0,0 +1,54 @@
+package pika.ui;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Storage { //Storage class used to handle the writing of the file after each actions
+    private static Path txtPath;
+
+    /**
+     * Constructor for Storage Class
+     * @param path the filepath where the txt file will be/is created
```

Comment:
Perhaps a newline in between the description and parameters? I noticed the same issue in several other places too.